charts/ofed-driver/parent/values.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "type": "object",
+    "properties": {
+        "ofed-driver": {
+            "type": "object",
+            "properties": {
+                "image": {
+                    "type": "object",
+                    "description": "the image tag is with a format {driverVersion}-${OSName}${OSVer}-${Arch}, refer to nvidia available image tag: https://catalog.ngc.nvidia.com/orgs/nvidia/teams/mellanox/containers/doca-driver/tags",
+                    "properties": {
+                        "Arch": {
+                            "type": "string",
+                            "description": "Arch",
```

Comment:
architect of the OS ， available values：amd64, arm64

Reply:
done